Makefile

```diff
@@ -435,6 +435,14 @@ tests/foundry/%: KOMPILE = . ./kevm_pyk/venv-prod/bin/activate && kevm kompile
 tests/foundry/out:
 	cd $(dir $@) && forge build
 
+tests/foundry/clean:
```

Comment:
Call this `foundry-clean` or `test-foundry-clean`, and make sure to add it to the `.PHONY` targets at the top of the file.